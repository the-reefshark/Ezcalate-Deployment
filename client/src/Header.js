import React from 'react';
import { makeStyles } from '@material-ui/core/styles';
import AppBar from '@material-ui/core/AppBar';
import Toolbar from '@material-ui/core/Toolbar';
import Typography from '@material-ui/core/Typography';
import IconButton from '@material-ui/core/IconButton';
import AccountCircle from '@material-ui/icons/AccountCircle';

const useStyles = makeStyles((theme) => ({
  root: {
    flexGrow: 1,
  },
  menuButton: {
    marginRight: theme.spacing(2),
  },
  title: {
    flexGrow: 1,
    fontFamily: 'Stylish',
    fontSize: "40px"
  },
  Appbar:{
    background: "black",
    position: "fixed",
    paddingBottom: "15px",
    // 'linear-gradient(45deg, #FE6B8B 30%, #FF8E53 90%)'
    zIndex: theme.zIndex.drawer + 1 // trying to make header [pop out]
    
  }
}));

export default function MenuAppBar(props) {
  const classes = useStyles();

  return (
      <AppBar position="static" className= {classes.Appbar}>
       
        <Toolbar >
          <Typography variant="h6" className={classes.title}>
            Ezcalate
          </Typography>
         
            <div>
              <IconButton
                aria-label="account of current user"
                aria-controls="menu-appbar"
                aria-haspopup="true"
                // onClick={props.handleMenu}
                color="inherit"
              >
                <AccountCircle />
              </IconButton>
              {/* <Menu
                id="menu-appbar"
                anchorOrigin={{
                  vertical: 'top',
                  horizontal: 'right',
                }}
                keepMounted
                transformOrigin={{
                  vertical: 'top',
                  horizontal: 'right',
                }}
                open={open}
                onClose={handleClose}
              >
                <MenuItem onClick={props.handleProfile}>Profile</MenuItem>
                <MenuItem onClick={handleClose}>My account</MenuItem>
              </Menu> */}
            </div>
          
        </Toolbar>
      </AppBar>

  );
}