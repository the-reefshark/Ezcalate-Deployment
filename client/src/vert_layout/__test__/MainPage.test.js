import React from 'react'
import Header from '../../Header'

import renderer from 'react-test-renderer'

it("Header matches snapshot", () => {
    const tree = renderer.create(<Header></Header>).toJSON()
    expect(tree).toMatchSnapshot()
})