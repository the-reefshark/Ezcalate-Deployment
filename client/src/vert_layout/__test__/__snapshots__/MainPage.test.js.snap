// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header matches snapshot 1`] = `
<header
  className="MuiPaper-root MuiAppBar-root MuiAppBar-positionStatic MuiAppBar-colorPrimary makeStyles-Appbar-4 MuiPaper-elevation4"
>
  <div
    className="MuiToolbar-root MuiToolbar-regular MuiToolbar-gutters"
  >
    <h6
      className="MuiTypography-root makeStyles-title-3 MuiTypography-h6"
    >
      Ezcalate
    </h6>
    <div>
      <button
        aria-controls="menu-appbar"
        aria-haspopup="true"
        aria-label="account of current user"
        className="MuiButtonBase-root MuiIconButton-root MuiIconButton-colorInherit"
        disabled={false}
        onBlur={[Function]}
        onDragLeave={[Function]}
        onFocus={[Function]}
        onKeyDown={[Function]}
        onKeyUp={[Function]}
        onMouseDown={[Function]}
        onMouseLeave={[Function]}
        onMouseUp={[Function]}
        onTouchEnd={[Function]}
        onTouchMove={[Function]}
        onTouchStart={[Function]}
        tabIndex={0}
        type="button"
      >
        <span
          className="MuiIconButton-label"
        >
          <svg
            aria-hidden={true}
            className="MuiSvgIcon-root"
            focusable="false"
            viewBox="0 0 24 24"
          >
            <path
              d="M12 2C6.48 2 2 6.48 2 12s4.48 10 10 10 10-4.48 10-10S17.52 2 12 2zm0 3c1.66 0 3 1.34 3 3s-1.34 3-3 3-3-1.34-3-3 1.34-3 3-3zm0 14.2c-2.5 0-4.71-1.28-6-3.22.03-1.99 4-3.08 6-3.08 1.99 0 5.97 1.09 6 3.08-1.29 1.94-3.5 3.22-6 3.22z"
            />
          </svg>
        </span>
      </button>
    </div>
  </div>
</header>
`;
